README.md

```diff
@@ -4,68 +4,68 @@ jrnl [![Build Status](https://travis-ci.com/jrnl-org/jrnl.svg?branch=master)](ht
-vacation. A journal helps you to keep track of the things you get done and how
-you did them. Your imagination may be limitless, but your memory isn't. For
-personal use, make it a good habit to write at least 20 words a day. Just to
-reflect what made this day special, or why you haven't wasted it. For
-professional use, consider a text-based journal to be the perfect complement to
-your GTD todo list - a documentation of what and how you've done it.
+## In a Nutshell
 
-In a Nutshell
--------------
+To make a new entry, just enter
 
-To make a new entry, just type
+``` sh
+jrnl yesterday: Called in sick. Used the time to clean the house and write my
+book.
+```
 
-    jrnl yesterday: Called in sick. Used the time cleaning the house and writing my book.
-
-and hit return. `yesterday:` will be interpreted as a timestamp. Everything
-until the first sentence mark (`.?!`) will be interpreted as the title, the
-rest as the body. In your journal file, the result will look like this:
+`yesterday:` is  interpreted by `jrnl` as a timestamp. Everything until the
+first sentence ending (either `.`, `?`, or `!`) is interpreted as the title, and
+the rest as the body. In your journal file, the result will look like this:
 
     [2012-03-29 09:00] Called in sick.
-    Used the time cleaning the house and writing my book.
+    Used the time to clean the house and write my book.
 
-If you just call `jrnl`, you will be prompted to compose your entry - but you
-can also configure _jrnl_ to use your external editor.
+Entering `jrnl` without any arguments launches an external editor where you can
+write your entry. `jrnl` will generate a time stamp for the entry after you save
+and close the editor window.
 
-For more information, please read our [documentation](https://jrnl.sh/overview/).
+For more information, please read the
+[documentation](https://jrnl.sh/overview/).
 
 ## Contributors
 
 ### Maintainers
-Our maintainers help keep the lights on for the project. Please thank them if
-you like jrnl.
+
+Our maintainers help keep the lights on for the project:
+
  * Jonathan Wren ([wren](https://github.com/wren))
  * Micah Ellison ([micahellison](https://github.com/micahellison))
 
+Please thank them if you like `jrnl`!
+
 ### Code Contributors
-This project is made with love by the many fabulous people who have
-contributed. Jrnl couldn't exist without each and every one of you!
+
+This project is made with love by the many fabulous people who have contributed.
+`jrnl` couldn't exist without each and every one of you!
 
 <a href="https://github.com/jrnl-org/jrnl/graphs/contributors"><img
 src="https://opencollective.com/jrnl/contributors.svg?width=890&button=false"
 /></a>
 
-If you'd also like to help make jrnl better, please see our [contributing
+If you'd also like to help make `jrnl` better, please see our [contributing
 documentation](CONTRIBUTING.md).
 
-### Financial Backers
+## Financial Backers
```

Comment:
Let's keep this a third-level headline, so "Financial Backers" are part of "Contributors"